pages/express-relay/contract-addresses.mdx

```diff
@@ -1,138 +1,199 @@
-    },
-    {
-      name: "Wrapped ETH",
-      value: "0x4200000000000000000000000000000000000006",
-    },
-    {
-      name: "Permit2 Contract",
-      value: "0x000000000022D473030F116dDEE9F6B43aC78BA3",
-    },
-  ]}
-/>
-
-This list contains the addresses of the commonly used assets present in opportunities on the Mode network:
-
-<AddressTable
-  explorer={"https://explorer.mode.network/address/$ADDRESS"}
-  entries={[
-    { name: "WETH", value: "0x4200000000000000000000000000000000000006" },
-    { name: "USDC", value: "0xd988097fb8612cc24eeC14542bC03424c656005f" },
-    { name: "USDT", value: "0xf0F161fDA2712DB8b566946122a5af183995e2eD" },
-    { name: "WBTC", value: "0xcDd475325D6F564d27247D1DddBb0DAc6fA0a5CF" },
-    { name: "ezETH", value: "0x2416092f143378750bb29b79eD961ab195CcEea5" },
-    { name: "STONE", value: "0x80137510979822322193FC997d400D5A6C747bf7" },
-    { name: "wrsETH", value: "0xe7903B1F75C534Dd8159b313d92cDCfbC62cB3Cd" },
-    { name: "weETH.mode", value: "0x04C0599Ae5A44757c0af6F9eC3b93da8976c150A" },
-    { name: "M-BTC", value: "0x59889b7021243dB5B1e065385F918316cD90D46c" },
-    { name: "MODE", value: "0xDfc7C877a950e49D2610114102175A06C2e3167a" },
-  ]}
-/>
-
-</Tabs.Tab>
-<Tabs.Tab>
-Auction Server endpoint: https://per-staging.dourolabs.app/
-
-### Optimism Sepolia
-
-**Network Details:**
-
-<AddressTable
-  explorer={"https://optimism-sepolia.blockscout.com/address/$ADDRESS"}
-  entries={[
-    {
-      name: "Express Relay",
-      value: "0x2F968931d1B7326d2875E9500980211dcc535eE5",
-    },
-    {
-      name: "Opportunity Adapter Factory",
-      value: "0xfA119693864b2F185742A409c66f04865c787754",
-    },
-    {
-      name: "Network Id",
-      value: "11155420",
-    },
-    {
-      name: "Chain Id",
-      value: "op_sepolia",
-    },
-    {
-      name: "Public RPC",
-      value: "https://sepolia.optimism.io/",
-    },
-    {
-      name: "Wrapped ETH",
-      value: "0x74A4A85C611679B73F402B36c0F84A7D2CcdFDa3",
-    },
-    {
-      name: "Permit2 Contract",
-      value: "0x000000000022D473030F116dDEE9F6B43aC78BA3",
-    },
-  ]}
-/>
-
-This list contains the addresses of the commonly used assets present in opportunities on the Optimism Sepolia network:
-
-<AddressTable
-  explorer={"https://optimism-sepolia.blockscout.com/address/$ADDRESS"}
-  entries={[
-    {
-      name: "BTC",
-      value: "0x3745007F7C8DD8Bec89b3B35f33f13f58b008533 ",
-    },
-    {
-      name: "USDC",
-      value: "0x1e3d75F24296abBC7bd10D151F51d758bCE379Ef",
-    },
-    {
-      name: "DOGE",
-      value: "0x0160Ba0A02A910e27b4189040aC74febcc476687 ",
-    },
-    {
-      name: "SOL",
-      value: "0x193aD24BeeDfAf27f217395f52fA20C7A36D79B3",
-    },
-    {
-      name: "PYTH",
-      value: "0x20e0b5Ff7aa971dAd8f489112Ae009A70608d2D5",
-    },
-  ]}
-/>
-
-</Tabs.Tab>
+<Tabs items={['EVM', 'SVM']}>
+  <Tabs.Tab>
+    <Tabs items={['Mainnet', 'Testnet']}>
+
+    <Tabs.Tab>
```

Comment:
I suggest collapsing these two nested tabs to one level with the following:
`Optimism Sepolia`, `Mode`, `Solana Mainnet`